src/modules/dashboard/components/CoinMachine/Confetti/Confetti.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useState } from 'react';
+import ReactConfetti from 'react-confetti';
+import { bigNumberify } from 'ethers/utils';
+
+import { Address } from '~types/index';
+import useWindowSize from '~utils/hooks/useWindowSize';
+
+import { useCurrentPeriodTokensQuery } from '~data/index';
+
+type Props = {
+  colonyAddress: Address;
+};
+
+const displayName = 'dashboard.CoinMachine.Confetti';
+
+const Confetti = ({ colonyAddress }: Props) => {
+  const [show, setShow] = useState<boolean>(false);
+  const dimensions = useWindowSize();
+
+  const { data: periodTokensData, refetch } = useCurrentPeriodTokensQuery({
+    variables: { colonyAddress },
+  });
+
+  useEffect(() => {
+    const timer = setInterval(() => {
+      refetch();
+    }, 1000);
+    if (show) {
+      clearInterval(timer);
+    }
+    return () => clearInterval(timer);
+  }, []);
```

Comment:
Shouldn't `refetch` and `show` be added as dependencies here ?

Reply:
Nope, we want fire it once

Reply:
Oh. In that case please add an `// eslint-ignore` declaration before so that the linter won't complain